src/twisted/scripts/_twistd_unix.py

```diff
@@ -68,7 +69,7 @@ class ServerOptions(app.ServerOptions):
     def opt_version(self):
         """Print version information and exit.
         """
-        print('twistd (the Twisted daemon) %s' % copyright.version)
+        print('twistd (the Twisted daemon) {}'.format(copyright.version))
```

Comment:
Why change all this formatting from `%` to `.format`?  It isn't necessary for this change and makes the diff harder to read. Worst of all, though, some of these changes affect lines that have no test coverage, so we won't know if they've introduced a regression.  I read through them and am pretty sure they don't, but I would feel more confident if this PR did not include any string formatting changes.